documentation/specs/DisplayTable.md

```diff
@@ -0,0 +1,238 @@
+# `DisplayTable` Component Specification
+
+## Overview
+
+A `DisplayTable` is used for presenting information displayed across columns and rows.
+
+---
+
+## Design
+
+`DisplayTable` is for presentational concerns only. It doesn't support interactions such as selection, sorting, and row expansion. See the `Table` component for an interactive table accommodating advanced use cases.
+
+Without interaction, complexities are shifted to surfacing a simple component API, structuring the table for style requirements, and ensuring appropriate application of accessibility markup.
+
+`DisplayTable` will use React Aria's `useTable` under the hood to reliably generate accessibile markup for the table.
+
+`DisplayTable` will use `columns` and `rows` as the primary mechanism for passing data to the table. While this is a slight departure from our dot-notation pattern used throughout the rest of Easy UI, this makes for a consistent data input pattern with our `Table`-related components.
+
+### API
+
+```ts
+type DisplayTableProps = AriaLabelingProps & {
+  /** Columns for the table. */
+  columns: Column[];
+
+  /**
+   * Horizontal alignment of data within their cells.
+   * @default start
+   */
+  dataAlignment?: "start" | "center" | "end";
```

Comment:
**Thought:** In a future task, perhaps we could try to create a pattern for naming for our alignment props for a simpler DX. I've been finding myself having to reference the documentation multiple times to get the right prop name when using the stack components. We have a few that are used for various purposes (`align`, `alignInline`, `alignBlock`, `alignItems`) but thinking there could be an opportunity to simplify as `halign`/`valign` or similar.



align | Horizontal alignment of children
-- | --




align | Horizontal alignment of children
-- | --

Reply:
i agree. made a task to followup